packages/skeleton/src/lib/utilities/Drawer/Drawer.svelte

```diff
@@ -66,7 +66,7 @@
 
 	// Props (transition)
 	/** Set the transition duration in milliseconds. */
-	export let duration: number = 200;
+	export let duration = 200;
```

Comment:
Be mindful to keep your changes limited to the scope of the issue being solved. While it's easy to make quick changes this in the code - this can be detrimental to reviews and merging. We'll let this slide for now since it's a useful change.

Reply:
Yeah I was going to mention this, this was done by the `pnpm ci:lint` command. It seems like it did it again when I quickly staged changes.
Is it possible that this has already been fixed in a newer release?